apps/avifenc.c

```diff
@@ -1117,6 +1118,8 @@ int main(int argc, char * argv[])
         } else if (!strcmp(arg, "-V") || !strcmp(arg, "--version")) {
             avifPrintVersions();
             goto cleanup;
+        } else if (!strcmp(arg, "-n") || !strcmp(arg, "--no-overwrite")) {
```

Comment:
I would rather not have the short "-n" version as I think its meaning is far from obvious, and it's not a flag that will be used very often, the long version is probably enough?

Reply:
If you want to drop the short "-n", that's not that big a deal to me. But I make use of this option more often than not (I had made a wrapper script for avifenc to do this, before making this PR). In my opinion --no-overwrite should be the default behavior, but I figured a change in default behavior would be less likely to be accepted, so I went with this.

FYI, for ffmpeg, "-n" has the same meaning as here.

Reply:
FYI I checked a few other tools: ffmpeg, imagemagick, cwebp, and cjxl. ffmpeg is the only one that does not overwrite by default. The other ones just overwrite and don't provide any flags. I do think it's a useful flag to have though, but I'm wondering if we might have a better use for -n in the future. I don't have a very strong opinion though.

Reply:
OK , I agree. Let's drop "-n". What tipped me over to the other side is the fact that there is already a "-y" flag short for "--yuv". So, if "-n" means "no", then people might assume "-y" means "yes".

What about shorting the flag to "--no"? I don't think there will come another feature where we will want to use just "--no" to represent.

I don't have a strong opinion on this either -- the only benefit is a little less typing :)